src/sql/workbench/parts/notebook/models/cell.ts

```diff
@@ -452,7 +452,11 @@ export class CellModel implements ICellModel {
 						let host = endpoint && endpoint.ipAddress ? endpoint.ipAddress : model.activeConnection.serverName;
 						let port = endpoint && endpoint.port ? ':' + endpoint.port.toString() : defaultPort;
 						let html = result.data['text/html'];
+						// Old Spark link
```

Comment:
What does "old" refer to here? Can we add an issue to track removing this in the future?

Reply:
CTP 3.1 or older. Will fix